misk/src/main/kotlin/misk/web/BoundAction.kt

```diff
@@ -111,7 +110,6 @@ internal class BoundAction<A : WebAction>(
     val initialSeedData = mapOf<Key<*>, Any?>(
       keyOf<HttpServletRequest>() to request,
       keyOf<HttpCall>() to httpCall,
-      keyOf<HttpRequest>() to httpCall,
```

Comment:
If this is set here, does it error now with the changes to ActionScope?

Reply:
nope, I'm only falling back to keys for subclasses if the key for the requested type is not set, so if a HttpRequest key is set, that'd still be returned.